### DISCUSSION TITLE
Docker pre-install question

### DISCUSSION THREAD

**Discussion by mellowism** [tteck/Proxmox#893] [Q&A]
Hi, 

I hope it's alright if I ask a question before trying it myself. 
I've used some of your helper scripts with great success before, so I am planning on rebuilding my server using some of your scripts later this evening. 

Last night after I updated Proxmox to 7.3-4 my docker installations all failed to start (see [here ](https://forum.proxmox.com/threads/since-pve-7-3-4-update-all-docker-container-disappeared-from-lxc-containers.120123/) for more info) 

Will your docker LXC installation script take this in account, or is it perhaps not relevant at all? 

One solution seems to be to add [VFS as storage driver](https://forum.proxmox.com/threads/docker-failed-to-register-layer-applylayer-exit-status-1-stdout-stderr-unlinkat-var-log-apt-invalid-argument.119954/) manually.

Thanks in advance!

**Comment by tteck** [comment]
For ZFS, use "Advanced" settings and select "Fuse Overlayfs"

**Reply by mellowism** [reply]
Perfect, thanks for a quick response! I will try it out in a bit.

**Comment by tteck** [comment]
This was discussed, and a solution was found for ZFS (ext4 not affected)
https://github.com/tteck/Proxmox/discussions/869

I did notice that the beta/test repo has new zfs files, so maybe Proxmox came up with their own solution. 🤷🏻‍♂️ 

![Screenshot 2023-01-03 10 05 56 AM](https://user-images.githubusercontent.com/21342670/210384274-9ce6e4a3-a2a7-4972-ab80-56100faed460.png)
